### PR TITLE
Add effects for Dazzling Rosary, Necrotic Cap and Wemmuth Trinket

### PR DESCRIPTION
**Pull request by pedrogrullada**

### CODE REVIEW

File: packs/equipment-effects/effect-necrotic-cap.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+    "_id": "wQDfwrrvfRxwG5CY",
+    "img": "icons/consumables/mushrooms/umbontae-bumpy-purple.webp",
+    "name": "Effect: Necrotic Cap",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Lesser)]{Necrotic Cap}</p>\n<p>Your @UUID[Compendium.pf2e.spells-srd.Item.Acid Grip] spell loses the acid trait, gains the fungus trait, and all acid damage the spell deals becomes void damage.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-necrotic-cap.json:7]: Consider linking to all variants of the Necrotic Cap. For an example, see Effect: Shining Symbol

Comment (pedrogrullada) [reply packs/equipment-effects/effect-necrotic-cap.json]: Consider it considered.

File: packs/equipment-effects/effect-dazzling-rosary.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+    "_id": "ryz7DdJTTXEgj9jf",
+    "img": "icons/commodities/gems/pearls-white.webp",
+    "name": "Effect: Dazzling Rosary",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dazzling Rosary] and @UUID[Compendium.pf2e.equipment-srd.Item.Dazzling Rosary (Greater)]</p>\n<p>When your @UUID[Compendium.pf2e.spells-srd.Item.Spiritual Weapon] critically Strikes a target, it becomes Dazzled until the beginning of your next turn.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-dazzling-rosary.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dazzling Rosary], @UUID[Compendium.pf2e.equipment-srd.Item.Dazzling Rosary (Greater)]</p>\n<p>When your @UUID[Compendium.pf2e.spells-srd.Item.Spiritual Weapon] critically Strikes a target, it becomes Dazzled until the beginning of your next turn.</p>"
```

File: packs/equipment-effects/effect-necrotic-cap.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
+{
+    "_id": "wQDfwrrvfRxwG5CY",
+    "img": "icons/consumables/mushrooms/umbontae-bumpy-purple.webp",
+    "name": "Effect: Necrotic Cap",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Lesser)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Moderate)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Greater)], and @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Major)]</p>\n<p>Your @UUID[Compendium.pf2e.spells-srd.Item.Acid Grip] spell loses the acid trait, gains the fungus trait, and all acid damage the spell deals becomes void damage.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-necrotic-cap.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Lesser)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Moderate)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Greater)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Major)]</p>\n<p>Your @UUID[Compendium.pf2e.spells-srd.Item.Acid Grip] spell loses the acid trait, gains the fungus trait, and all acid damage the spell deals becomes void damage.</p>"
```

File: packs/equipment-effects/effect-dazzling-rosary.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
+{
+    "_id": "ryz7DdJTTXEgj9jf",
+    "img": "icons/commodities/gems/pearls-white.webp",
+    "name": "Effect: Dazzling Rosary",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dazzling Rosary], @UUID[Compendium.pf2e.equipment-srd.Item.Dazzling Rosary (Greater)]</p>\n<p>When your @UUID[Compendium.pf2e.spells-srd.Item.Spiritual Weapon] critically Strikes a target, it becomes Dazzled until the beginning of your next turn.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-dazzling-rosary.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dazzling Rosary], @UUID[Compendium.pf2e.equipment-srd.Item.Dazzling Rosary (Greater)]</p>\n<p>When your <em>spiritual weapon</em> critically Strikes a target, it becomes Dazzled until the beginning of your next turn.</p>"
```

File: packs/equipment-effects/effect-necrotic-cap.json [hunk 5]
```diff
@@ -0,0 +1,71 @@
+{
+    "_id": "wQDfwrrvfRxwG5CY",
+    "img": "icons/consumables/mushrooms/umbontae-bumpy-purple.webp",
+    "name": "Effect: Necrotic Cap",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Lesser)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Moderate)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Greater)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Major)]</p>\n<p>Your @UUID[Compendium.pf2e.spells-srd.Item.Acid Grip] spell loses the acid trait, gains the fungus trait, and all acid damage the spell deals becomes void damage.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-necrotic-cap.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Lesser)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Moderate)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Greater)], @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Major)]</p>\n<p>Your <em>acid grip</em> spell loses the acid trait, gains the fungus trait, and all acid damage the spell deals becomes void damage.</p>"
```

File: packs/equipment-effects/effect-wemmuth-trinket.json [hunk 6]
```diff
@@ -0,0 +1,51 @@
+{
+    "_id": "mS82CkuoGfOfQ9Jt",
+    "img": "icons/commodities/treasure/broach-spiral-blue.webp",
+    "name": "Effect: Wemmuth Trinket",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Wemmuth Trinket]</p>\n<p>Creatures who fail or critically fail their Reflex saves against @UUID[Compendium.pf2e.spells-srd.Item.Entangling Flora] also take piercing damage equal to the spell's rank.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-wemmuth-trinket.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Wemmuth Trinket]</p>\n<p>Creatures who fail or critically fail their Reflex saves against <em>entangling flora</em> also take piercing damage equal to the spell's rank.</p>"
```

**Pull request merged**